packages/plugin-client-drizzle/test/integration.ava.ts

```diff
@@ -0,0 +1,2187 @@
+
+  await db.insert(usersDistinctTable).values([
+    { id: 1, name: 'John' },
+    { id: 1, name: 'John' },
+    { id: 2, name: 'John' },
+    { id: 1, name: 'Jane' }
+  ]);
+  const users1 = await db
+    .selectDistinct()
+    .from(usersDistinctTable)
+    .orderBy(usersDistinctTable.id, usersDistinctTable.name);
+  const users2 = await db
+    .selectDistinctOn([usersDistinctTable.id])
+    .from(usersDistinctTable)
+    .orderBy(usersDistinctTable.id);
+  const users3 = await db
+    .selectDistinctOn([usersDistinctTable.name], { name: usersDistinctTable.name })
+    .from(usersDistinctTable)
+    .orderBy(usersDistinctTable.name);
+
+  await db.execute(sql`drop table ${usersDistinctTable}`);
+
+  t.deepEqual(users1, [
+    { id: 1, name: 'Jane' },
+    { id: 1, name: 'John' },
+    { id: 2, name: 'John' }
+  ]);
+
+  t.deepEqual(users2.length, 2);
+  t.deepEqual(users2[0]?.id, 1);
+  t.deepEqual(users2[1]?.id, 2);
+
+  t.deepEqual(users3.length, 2);
+  t.deepEqual(users3[0]?.name, 'Jane');
+  t.deepEqual(users3[1]?.name, 'John');
+});
+
+test.skip('insert returning sql', async (t) => {
+  const { db } = t.context;
+
+  const users = await db
+    .insert(usersTable)
+    .values({ name: 'John' })
+    .returning({
+      name: sql`upper(${usersTable.name})`
+    });
+
+  t.deepEqual(users, [{ name: 'JOHN' }]);
+});
+
+test.skip('delete returning sql', async (t) => {
+  const { db } = t.context;
+
+  await db.insert(usersTable).values({ name: 'John' });
+  const users = await db
+    .delete(usersTable)
+    .where(eq(usersTable.name, 'John'))
+    .returning({
+      name: sql`upper(${usersTable.name})`
+    });
+
+  t.deepEqual(users, [{ name: 'JOHN' }]);
+});
+
+test.skip('update returning sql', async (t) => {
+  const { db } = t.context;
+
+  await db.insert(usersTable).values({ name: 'John' });
+  const users = await db
+    .update(usersTable)
+    .set({ name: 'Jane' })
+    .where(eq(usersTable.name, 'John'))
+    .returning({
+      name: sql`upper(${usersTable.name})`
+    });
+
+  t.deepEqual(users, [{ name: 'JANE' }]);
+});
+
+test.skip('update with returning all fields', async (t) => {
+  const { db } = t.context;
+
+  const now = Date.now();
+
+  await db.insert(usersTable).values({ name: 'John' });
+  const users = await db.update(usersTable).set({ name: 'Jane' }).where(eq(usersTable.name, 'John')).returning();
+
+  t.assert(users[0]!.createdAt instanceof Date);
+  t.assert(Math.abs(users[0]!.createdAt.getTime() - now) < 100);
+  t.deepEqual(users, [{ id: 1, name: 'Jane', verified: false, jsonb: null, createdAt: users[0]!.createdAt }]);
+});
+
+test.skip('update with returning partial', async (t) => {
+  const { db } = t.context;
+
+  await db.insert(usersTable).values({ name: 'John' });
+  const users = await db.update(usersTable).set({ name: 'Jane' }).where(eq(usersTable.name, 'John')).returning({
+    id: usersTable.id,
+    name: usersTable.name
+  });
+
+  t.deepEqual(users, [{ id: 1, name: 'Jane' }]);
```

Comment:
The `id` is not `1` but rather the Xata generated id that has a format `rec_{randomstring}`.